pkg/sysregistriesv2/shortnames.go

```diff
@@ -34,15 +34,9 @@ func shortNameAliasesConfPath(ctx *types.SystemContext) (string, error) {
 	}
 
 	// Rootless user
-	var cacheRoot string
-	if xdgCache := os.Getenv("XDG_CACHE_HOME"); xdgCache != "" {
-		cacheRoot = xdgCache
-	} else {
-		configHome, err := homedir.GetConfigHome()
-		if err != nil {
-			return "", err
-		}
-		cacheRoot = filepath.Join(configHome, ".cache")
+	cacheRoot, err := homedir.GetCacheHome()
```

Comment:
This cannot compile yet.

You need to update the dependency on `c/storage`.  Since @rhatdan cut a new release with your commit there, you can now do a `go get github.com/containers/storage@v1.26.0` and add the changes to this commit.